twittermap/public/javascripts/map/controllers.js

```diff
@@ -447,7 +462,10 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
       }
 
       function setNormalizedCount(geo, r){
-        geo['properties']['count'] = r['count'] / r['population'] * cloudberryConfig.normalizationUpscaleFactor;
+        var normalizedCount = r['count'] / r['population'] * cloudberryConfig.normalizationUpscaleFactor;
+        geo['properties']['count'] = normalizedCount;
+        if(normalizedCount > normalizedCountMax)  // update max to enable dynamic legends
+          normalizedCountMax = normalizedCount;
```

Comment:
Are you saying the `'count'` property for each geo? If so, we don't. We only need to keep track of the maximum count, and this is what this line of code is doing.